javascript-crash-course.md

```diff
@@ -72,8 +73,9 @@ We will gather to hear the talk JavaScript is a Buffet, not a Main Course by [Ch
 
 
 Talk Description: 
-Now that you learned about JavaScript, you're ready to take on the world as JavaScript runs everywhere.
-The trick is to not get overwhelmed and to find what makes you happy as happy developers are effective developers. Chris Heilmann has a 18 year old history of using the language and wrote many books about it. Let him help you not get lost in the rabbit hole that is JavaScript in 2017.
+Now that you learned about JavaScript, you're ready to take on the world as JavaScript runs everywhere.The trick is to not get overwhelmed and to find what makes you happy as happy developers are effective developers. 
```

Comment:
whitespace between ".The"  is missing

Reply:
Good point - but it seems to be with space on the website: http://wtmberlin.com/javascript-crash-course/

Reply:
Oh @emedvedev fixed it 👍